initializers/model.go

```diff
@@ -0,0 +1,11 @@
+package initializers
+
+import (
+	"github.com/beego/beego/v2/client/orm"
+	"github.com/beego/beego/v2/client/orm/filter/bean"
+)
+
+func SetModelDefaultValueFilter() {
```

Comment:
added this to be able to set [default value](https://beego.me/docs/mvc/model/models.md#default-value) for `Result` [status](https://github.com/carryall/go-google-scraper-challenge/pull/42/files#diff-e5575e255237767a833fc7e25a58186e9e03157369c630efd4f479e07f5029caR19)